youtrack_sdk/client.py

```diff
@@ -36,37 +36,39 @@ def _build_url(
         *,
         path: str,
         fields: Optional[str] = None,
+        query: Optional[str] = None,
         offset: Optional[int] = None,
         count: Optional[int] = None,
     ) -> str:
-        query = "&".join(
+        query_parameters = "&".join(
             tuple(
                 f"{key}={value}"
                 for key, value in {
                     "fields": fields,
+                    "query": query,
```

Comment:
I think we should think carefully before adding an additional parameter to this function. Other API methods may have more additional parameters that can be used only in that method or in a several methods. For example, [update issue](https://www.jetbrains.com/help/youtrack/devportal/operations-api-issues.html#update-Issue-method) has `muteUpdateNotifications` parameter. Instead of adding extra parameter to `_build_url` and `_send_request`, I would add `**extra_query_args` (or better name) and combine it with a dict of common parameters (`fields`, `$skip` and `$top`). It means that any method can provide additional query arguments to request even if they are used only in that method.

Reply:
You are right, PR updated